testproject/Assets/Tests/Manual/NetworkAnimatorTests/AnimationUserInput.cs

```diff
@@ -26,15 +26,21 @@ private void LateUpdate()
             {
                 foreach (var controller in m_Controllers)
                 {
-                    controller.ToggleRotateAnimation();
+                    if ((controller.IsServerAuthoritative && (IsServer || IsOwner)) || (!controller.IsServerAuthoritative && IsOwner))
```

Comment:
Just checking. when `IsOwner` and `IsServerAuthoritative` we still want this ?

Reply:
Actually, all of that is not needed when checking for user input. The conditional requirement to check for user input was culled down to:
- IsSpawned has to be true
- IsOwner has to be true
If neither of the above, then it exist early.  Otherwise, it checks for any user-owner input.